src/orchard/java/resource.clj

```diff
@@ -0,0 +1,52 @@
+(ns orchard.java.resource
+  "Resolve JVM resource-related paths."
```

Comment:
It's not really clear what's the difference between the this ns and `orchard.resource`, as both of them are Clojure/Java specific right now.

Reply:
Ok so maybe there was some overengineering on my part, I can squash them together. It felt tidier to have one ns for the entry point of the op only.

Reply:
I'm fine with a separation if it's clear what the purpose is - e.g. if the other namespace is something we intend to use to support CLR down the road. Right now they just seemed confusing. 

There's also the fact that we should be designing an optimal API here for general use, not just reflect how things were in `cider-nrepl`. It's fine for the op to have to call a couple of functions instead of one if that would result in a better Orchard API in general.

Reply:
I will bring everything in `orchard.resource` for now but I still think we need a better API for this.